app/src/org/commcare/utils/SyncDetailCalculations.java

```diff
@@ -152,12 +152,13 @@ private static boolean unsentFormNumberLimitExceeded(int numUnsentForms) {
 
     private static boolean unsentFormTimeLimitExceeded(long lastSyncTime) {
         SharedPreferences prefs = CommCareApplication.instance().getCurrentApp().getAppPreferences();
-        int unsentFormTimeLimit = Integer.parseInt(prefs.getString(UNSENT_FORM_TIME_KEY, "5"));
+        double unsentFormTimeLimitInDays = Double.parseDouble(prefs.getString(UNSENT_FORM_TIME_KEY, "5"));
+        int unsentFormTimeLimitInMinutes = (int)(unsentFormTimeLimitInDays * 24 * 60);
```

Comment:
How about converting this to seconds instead of minutes for even better precision?

Reply:
or miliseconds?
that way you avoid any conversion in the comparison later.

Reply:
I battled with the same question myself and ended up settling with minutes.